app/lib/foreman_deployments/tasks/search_task_definition.rb

```diff
@@ -7,6 +7,26 @@ def run
           results = SerializableArray.new(results)
           SearchTaskDefinition.create_output(results, output)
         end
+
+        def self.dereference_output(storage, path)
```

Comment:
It's the method that is used by dynflow patch - basically it will be called on deserialization process of dynflow task parameters. First argument (`storage`) is the serialized task output and the second one is a path to the relevant property in the output of the task.
Let's wait for Ivan's comments on this, if he approves the solution I'll add tests.